production/terraform/modules/s3/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "us-east-2"
+}
+
+variable "bucket_name" {
+  description = "Bucket name for Loki storage"
+  type    = string
+  default = "loki-data"
+}
+
+variable "cluster_name" {
+  description = "Name of EKS cluster"
+  type = string
+}
+
+variable "namespace" {
+  description = "Namespace of Loki installation"
+  type        = string
+  default     = "default"
```

Comment:
I think that we need to make the user define each of these variables... otherwise, it will be a source of errors when the user deployed it but forgot to specify the correct cluster name or namespace or anything else....